starky/src/stark.rs

```diff
@@ -44,11 +44,13 @@ pub trait Stark<F: RichField + Extendable<D>, const D: usize>: Sync {
     /// constraints over `F`.
     fn eval_packed_generic<FE, P, const D2: usize>(
         &self,
-        vars: &Self::EvaluationFrame<FE, P, D2>,
-        yield_constr: &mut ConstraintConsumer<P>,
+        _vars: &Self::EvaluationFrame<FE, P, D2>,
+        _yield_constr: &mut ConstraintConsumer<P>,
     ) where
         FE: FieldExtension<D2, BaseField = F>,
-        P: PackedField<Scalar = FE>;
+        P: PackedField<Scalar = FE>,
+    {
+    }
```

Comment:
To prevent any risk of misuse for downstream users of `starky`, I think we shouldn't have the blanket impl do this, and keep it as it was. If one (like in our usecase) was to need such unconstrained table, they would need to explicitly mention it in their implementation.